tests/actionsFactory.spec.ts

```diff
@@ -28,9 +26,9 @@ describe("actionsFactory test", () => {
     describe("create command action", () => {
         it("suceeds with params", () => {
             const actionJson = {
-                [ActionJsonKey.ActionType]: ActionType.Command,
-                [ActionJsonKey.CommandName]: "myCommand",
-                [ActionJsonKey.CommandParams]: ["param1", "param2"]
+                actionType: "COMMAND",
```

Comment:
Please evaluate the usefulness of declaring the type of this const
```
const actionJson: ICommadAction ...
```

Reply:
In general for **all** tests.

e.g maintainability? refactoring (new property added) ect...

Reply:
actionJson is a json file, not an ICommandAction object